core/config/docs/chains-solana.toml

```diff
@@ -40,6 +40,10 @@ BlockHistoryPollPeriod = '5s' # Default
 ComputeUnitLimitDefault = 200_000 # Default
 # EstimateComputeUnitLimit enables or disables compute unit limit estimations per transaction. If estimations return 0 used compute, the ComputeUnitLimitDefault value is used, if set.
 EstimateComputeUnitLimit = false # Default
+# BlockHistorySize is the number of blocks to take into consideration when using FeeEstimatorMode = 'blockhistory' to determine compute unit price.
+# If set to 1, LatestBlockEstimator will be used. The compute unit price will be determined by the median of the last block's compute unit prices. 
+# If set N > 1, MultipleBlocksEstimator will be used. The compute unit price will be determined by the average of the medians of the last N blocks' compute unit prices.
+BlockHistorySize = 1 # Default
```

Comment:
Think it might be better to move this config under `BlockHistoryPollPeriod` so the docs and examples for BHE configs are next to each other. Feel like that might help people notice configs related to the same component more easily